core/src/main/java/jenkins/model/GlobalConfiguration.java

```diff
@@ -99,19 +99,10 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
         @SuppressWarnings("rawtypes")
         @Override
         public boolean filter(Object context, Descriptor descriptor) {
-            boolean result = false;
-            try {
-                if (descriptor instanceof GlobalConfiguration) {
-                    return Functions.hasPermission(((GlobalConfiguration) descriptor).getPermission());
-                } else {
-                    return true;
-                }
-            } catch (AccessDeniedException e) {
-                return false;
-            } catch (ServletException | IOException e) {
-                e.printStackTrace();
-                return true;
+            if (descriptor instanceof GlobalConfiguration) {
+                return Jenkins.get().hasPermission(((GlobalConfiguration) descriptor).getPermission());
```

Comment:
I think you still need to filter with `DescriptorVisibilityFilter`

Reply:
will address in a separate PR if needed

Reply:
its a change in the contract and who knows who is filtering GlobalConfiguration in some random closed source plugin.
the change isn't here - but the caller that no longer filters calls other DescriptorVisibilityFilter.

Reply:
this is to merge some minor fix in the experimental-manage-permission PR.
We will fix this filtering issue you are pointing in a different PR into the same branch soon